supported_build_configurations.yaml

```diff
@@ -61,6 +61,14 @@ test-pid:
 #  - gimbtonomy
 #  - science
 
+test-pid-tuning-api:
+  - nucleo
+  - arm
+#  - gimbtonomy
```

Comment:
I'd rather not commit more "this should work with this board but it doesn't" configs. could u get them to compile for these boards?